.github/workflows/ci.yml

```diff
@@ -238,7 +238,7 @@ jobs:
           pytest --suppress-no-test-exit-code --cov=docarray --cov-report=xml \
             -v -s -m "not gpu" ${{ matrix.test-path }}
           echo "::set-output name=codecov_flag::docarray"
-        timeout-minutes: 30
+        timeout-minutes: 40
```

Comment:
why is this needed now?

Reply:
Currently `tests/unit/array/mixins/oldproto` takes more than 30 mins to finish